### DISCUSSION TITLE
Skipping eligible roles as required Graph permission 'RoleEligibilitySchedule.ReadWrite.Directory' was not present.

### DISCUSSION THREAD

**Discussion by simon-barratt-seq** [maester365/maester#277] [General]
I have just started seeing this warning popup. Any ideas? Not sure why it would require ReadWrite permission.

![image](https://github.com/maester365/maester/assets/152890576/78f8984d-157c-4290-bc07-036a2cdd3ddc)

**Comment by soulemike** [comment]
Hi Simon, there is a need for ReadWrite to retrieve members of eligible role assignments. There is more detail here too. 

https://github.com/maester365/maester/issues/195#issuecomment-2135816410

**Reply by simon-barratt-seq** [reply]
Thanks. The app registration does have that particular permission, but it still throws the warning.

**Reply by soulemike** [reply]
If you are using `Connect-Maester` are you including the -Privileged parameter?

**Reply by simon-barratt-seq** [reply]
I am now! Thanks that fixed it.

**Reply by soulemike** [reply]
Nice, glad that solved it. I know we just recently merged in some new docs around that too, so it wasn't the most obvious "feature".
